### ISSUE TITLE
Version 1.1.6 Does not Install

### ISSUE THREAD

**Issue by ericellb** [maykar/plex_assistant#107]
I know this is a bad issue report and will most likely get closed.

I tried installing this integration on 1.1.6, and the screen just hangs when adding the integration. (Already added to HACS, only adding it to HA Integrations).

I downgraded to 1.1.15 and it worked no issue, did not have time to debug with changing HA log levels, so just a heads up this build might be broken.

**Comment by maykar** [comment]
This is one where any more information is helpful. Did you restart after upgrade? Please, provide logs when possible.

**Comment by ericellb** [comment]
I restarted after originally adding it from HACS.

I know more info would be appreciated :( It was pretty late and my first gut feeling was to downgrade since I saw the latest release was 2 days old, I didint really have the time to further debug the issue.

As I mentioned pretty shitty bug report, just wanted to bring it up in the case that anyone else has this issue, hopefully they could chime in or have a reference if they are also having issues.

Great tool though, love it!
